tests/ssl-opt.sh

```diff
@@ -726,6 +742,28 @@ run_test() {
         esac
     fi
 
+    # update CMD_IS_GNUTLS variable
+    is_gnutls "$SRV_CMD"
+
+    # if the server uses gnutls but doesn't set priority, explicitly
+    # set the default priority
+    if [ "$CMD_IS_GNUTLS" -eq 1 ]; then
+        if ! echo "$SRV_CMD" | grep '\-\-priority' >/dev/null; then
+            SRV_CMD="$SRV_CMD $G_PRI"
+        fi
```

Comment:
Please use `case` in preference to `grep`: it has fewer quoting gotchas and it's faster.
```suggestion
        case "$SRV_CMD" in
              *--priority*) :;;
              *) SRV_CMD="$SRV_CMD $G_PRI";;
        esac
```
and likewise below